client/src/components/add-job-post-component/addJobPosting.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import dayjs from "dayjs";
+import AddJobLead from "./index";
+import UserType from "../../prop-types/UserType";
+
+function AddJobLeadParent({ currUser }) {
+  const initialState = {
```

Comment:
initialState is an object but in [client/src/components/add-job-post-component/index.jsx](https://github.com/uoftblueprint/tces/pull/122/files#diff-fd6b8b12c813f19587485e34512b350ba8315a430d0f832f1409454dee8ca09e) I think it's defined as an array. Might need to look at that but I'm not sure if that would cause an error.

jobPostData: PropTypes.array.isRequired,